examples/name_registry.vy

```diff
@@ -1,14 +0,0 @@
-
-registry: address[bytes[100]]
-
-
-@public
-def register(name: bytes[100], owner: address):
-    assert self.registry[name] == ZERO_ADDRESS  # check name has not been set yet.
-    self.registry[name] = owner
-
-
-@public
-@constant
-def lookup(name: bytes[100]) -> address:
-    return self.registry[name]
```

Comment:
This is a duplicate file, the test uses `examples/name_registry/name_registry.vy` which is more consistent with the file structure.